### PR TITLE
fix overencode; fix double-decode

### PR DESCRIPTION
**Pull request by forresto**
Switch to `he`

Also fix a double-decode bug where `&amp;amp;` would show as `&` instead of expected `&amp;`

### CODE REVIEW

**Comment by forresto** [comment]
hold this one... need to try something else

**Comment by bergie** [comment]
Note that we're using `entities` here since that is what [html-flatten](https://github.com/the-grid/html-flatten) uses as well

File: src/index.js [hunk 1]
```diff
@@ -76,7 +79,7 @@ export default class CEd {
     this.id = block.id
     let el = document.createElement('div')
     el.innerHTML = this.block.html
-    this.editor.setValue(decodeHTML(el.textContent))
+    this.editor.setValue(decode(el.textContent))
```
Comment (forresto) [inline src/index.js]: Do we need this `decode`? Seems like `el.textContent` would already be decoded.

Comment (forresto) [inline src/index.js]: This is a bug: turns `&amp;amp;` into `&` expected `&amp;`

**Comment by forresto** [comment]
@bergie #4 is desired behavior?

**Pull request merged**